docs/how-to-guides/data-basics/1-setup.md

```diff
@@ -0,0 +1,151 @@
+}
+
+window.addEventListener( 'load', function() {
+	wp.element.render(
+		wp.element.createElement( MyFirstApp ),
+		document.querySelector( '#my-first-gutenberg-app' )
+	);
+}, false );
+```
+
+**style.css:**
+```css
+.toplevel_page_my-first-gutenberg-app #wpcontent {
+	background: #FFF;
+}
+#my-first-gutenberg-app {
+	max-width: 500px;
+}
+#my-first-gutenberg-app ul,
+#my-first-gutenberg-app ul li {
+	list-style-type: disc;
+}
+#my-first-gutenberg-app ul {
+	padding-left: 20px;
+}
+#my-first-gutenberg-app .components-search-control__input {
+	height: 36px;
+	margin-left: 0;
+}
+```
+
+**my-first-gutenberg-app.php:**
+```php
+<?php
+/**
+ * Plugin Name: My first Gutenberg App
+ *
+ */
+
+function my_admin_menu() {
+	// Create a new admin page for our app
+	add_menu_page(
+		__( 'My first Gutenberg app', 'my-textdomain' ),
+		__( 'My first Gutenberg app', 'my-textdomain' ),
+		'manage_options',
+		'my-first-gutenberg-app',
+		function() {
+			echo '
+				<h2>Pages</h2>
+				<div id="my-first-gutenberg-app"></div>
+			';
+		},
+		'dashicons-schedule',
+		3
+	);
+}
+
+add_action( 'admin_menu', 'my_admin_menu' );
+
+function load_custom_wp_admin_scripts( $hook ) {
+	// Load only on ?page=my-first-gutenberg-app
+	if ( $hook !== 'toplevel_page_my-first-gutenberg-app' ) {
+		return;
+	}
+
+	// Load the required WordPress packages:
+
+	// wp-components is a library of generic WordPress components
+	// used for building consistent user interfaces across the board.
+	wp_enqueue_style( 'wp-components' );
+	wp_enqueue_script( 'wp-components' );
+
+	// wp-data provides data management backbone such as the Redux
+	// implementation or data resolution mechanisms.
+	wp_enqueue_script( 'wp-data' );
+
+	// wp-core-data is a glue between WordPress Core and wp-data.
+	// It provides a Redux store with a number of selectors and actions to
+	// power common tasks such as loading the data from the WordPress REST API,
+	// editing it in the browser, and persisting the changes back to the REST API.
+	wp_enqueue_script( 'wp-core-data' );
+
+	// Load our app.js
+	wp_register_script(
+		'my-first-gutenberg-app',
+		plugins_url( 'my-first-gutenberg-app/app.js' ),
+		array( 'wp-components', 'wp-data', 'wp-core-data' ),
+	);
+	wp_enqueue_script( 'my-first-gutenberg-app' );
+
+	// Load our style.css
+	wp_register_style( 'my-first-gutenberg-app', plugins_url( 'my-first-gutenberg-app/style.css' ) );
+	wp_enqueue_style( 'my-first-gutenberg-app' );
+}
+
+add_action( 'admin_enqueue_scripts', 'load_custom_wp_admin_scripts' );
+```
+
+If you now go to the Plugins page, you should see a plugin called **My first Gutenberg App**. Go ahead and activate it. A new menu item labeled _My first Gutenberg app_ should show up. Once you click it, you will see a page that says _Hello from JavaScript!_:
+
+![](./media/setup/hello-from-js.jpg)
+
+Congratulations! You are now ready to start building the app!
+
+## Readability vs Convenience
```

Comment:
I think you can skip this whole paragraph or link to some existing page that covers the same topic. 

A good reference would be also a tutorial that @getdave contributed for the standalone block editor:

https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/platform/custom-block-editor/tutorial.md

Those are advanced tutorials and they should be using the most efficient syntax so in my opinion the build step is essential here.

Reply:
I agree in principle, but I think of this tutorial as more of a first step towards understanding advanced Gutenberg concepts. It can benefit from a build tool but I don't think it should _require_ one. I'm thinking of myself here – I'm pretty impatient and I would instinctively look for a different tutorial with less pre-requisites. 

That being said, let's see if I can make this section more succinct.

Reply:
That's why we always try to pair the tutorial with https://github.com/WordPress/gutenberg-examples so you don't have to write a single line of code to run it 😄

Reply:
Ooh that's great, thank you for sharing! I'll look into that